workshops/test/test_person.py

```diff
@@ -119,7 +122,7 @@ def _check_person(self, doc, person):
                     .format(key, value, type(value), node.text)
 
     def _get_field(self, doc, key):
-        '''Get field from person display.'''
+        """ Get field from person display. """
```

Comment:
There are two commits, one fixing bug and one fixing PEP8 issues. Is that enough or do you want me to split this into two PRs?